index.html

```diff
@@ -139,13 +139,12 @@
         <div class="container7">
             <div class="row1">
                 <div class="abc"><a href="#"><i class="fa fa-2x fa-map-marker" class="contact"
-                            style="color: #F6FFFA;"></i></a></div>
+                            ></i></a></div>
                 <div class="def">Skibiddi toilet, Area 51</div>
                 <div class="abc"><a href="#"><i class="fa fa-2x fa-phone" class="contact"
-                            style="color: #F6FFFA;"></i></a></div>
+                            ></i></a></div>
                 <div class="def"> +91 4206969420</div>
-                <div class="abc"><a href="#"><i class="fa fa-2x fa-envelope" class="contact"
-                            style="color: #F6FFFA;"></i></a></div>
+                <div class="abc"><a href="#"><i class="fa fa-2x fa-en></i></a></div>
```

Comment:
hey @Akshat1276  there are few issues with your commits
firstly there is a missing ' " ' here and the class name is also not set properly (match with the actual code)